.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
Wait until the Issue Form PRs have been merged into `main`:

- [ ] https://github.com/testing-library/eslint-plugin-testing-library/pull/433
- [ ] https://github.com/testing-library/eslint-plugin-testing-library/pull/434
- [ ] https://github.com/testing-library/eslint-plugin-testing-library/pull/435